src/decoration/decoration.ts

```diff
@@ -1,7 +1,10 @@
+import { privateEncrypt } from 'crypto';
```

Comment:
this is what happens when one forget they're not in python for a moment

Reply:
That is 100% something that I added by accident. Not used anywhere, so we can get rid of it.